MarsQACompetitionTaskNUnit/Tests/CertificationTest.cs

```diff
@@ -0,0 +1,144 @@
+﻿using MarsQACompetitionTaskNUnit.Assertions;
+using NUnit.Framework;
+
+namespace MarsQACompetitionTaskNUnit.Tests
+{
+    [Parallelizable]
+    [TestFixture]
+    public class CertificationTest: BaseTest 
+    {
+        [SetUp]
+        public void SetUpCertification()
+        {           
+            certificationPageObject.NavigateToCertificationTab();
+            Thread.Sleep(1000);
+        }
+
+        [Test, Order(1), Description("This test create a new Certification record")]
+        public void TestCreateCertificationRecord()
+        {         
+            certificationPageObject.ClearCertification();
```

Comment:
so clear all data should be done once before all the tests execute, if test run parallel it will  wipe out other test data!

Reply:
Clear all data should be done once before all the tests execute